brave/brave-core
# Import p2p keys

First (Import) part of https://github.com/brave/brave-browser/issues/15566

- Added "Import" button to dialog that allows to import a key with predefined name
- Renamed button from Submit to Generate to be consistent with another button
- Implemented some refactoring because I reuse file uploads logic to ipfs node (Moved all network blobs constructions to ipfs_network_utils, there are some related changes in import classes. Finally refactoring will be finished in next PR in https://github.com/brave/brave-browser/issues/15758)
- Added alertion about error if import was not successful

![image](https://user-images.githubusercontent.com/2965009/117813125-60c99300-b26b-11eb-9896-49984b53b5ba.png)


![image](https://user-images.githubusercontent.com/2965009/117813194-750d9000-b26b-11eb-9da4-160281ff93fb.png)

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Try import keys
- Try import any other files, should show error
